### DISCUSSION TITLE
NEW: ColumnView - Fluent UI based Headless CMS

### DISCUSSION THREAD

**Discussion by sandra-burdy** [microsoft/fluentui#19796] [Show and tell]
Hi everyone,

We have just released [Burdy](https://burdy.io), a new Open Source Headless CMS that uses Fluent UI as a UI/Admin framework and is fully written in TypeScript. For our purposes, we have developed **ColumnView** to help authors structure Pages in a Hierarchical view.

You can see how it looks in the middle picture, we are using Fluent UI look and feel and Selection to be able to switch back and forth between Details List and Column View.

![burdy](https://user-images.githubusercontent.com/90346866/133385459-0013063b-6bea-436b-8d7f-5e4cd81b8ec2.png)

Is this something you would be interested in? We might be able to open Pull Request for ColumnView in a month or two as it would need a bit of a rework.

King regards,
S

**Comment by blendsdk** [comment]
Looks great 👍

**Comment by JustSlone** [comment]
Looks cool and glad you are finding good value in Fluent UI. We are heads down right now working on a major update to our component model and our components (Fluent UI vNext), so we're not able to accept major component contributions right now as we're stretched pretty thin. 

We do have aspirations to spin up a place where folks can more easily contribute and build on each other's community built components, but that's further out and not actually planned yet.
